src/java/org/apache/cassandra/cql3/conditions/AbstractConditions.java

```diff
@@ -32,11 +37,18 @@ public void addFunctionsTo(List<Function> functions)
     {
     }
 
+    @Override
     public Iterable<ColumnMetadata> getColumns()
     {
         return null;
     }
 
+    @Override
+    public Set<ColumnMetadata> getAnalyzedColumns(IndexRegistry indexRegistry)
```

Comment:
No, I don't. I don't remember what I was doing when I added that, possibly trying to pass the index itself instead of the index registry? I have removed the unused import.

Reply:
Yeah, removing the unused import was the right call. Thanks!